codestodians/food-basket-api
# [Feat: #150399214] Integrate Travis CI and coveralls

#### What does this PR do?
- Sets up a Travis Continuous Integration for commits and PRs.
- Sets up coveralls test coverage service.

#### How should this be manually tested?
- Check The recent Travis builds.
- Check Coverage Information on [coveralls.io](https://coveralls.io/github/codestodians/food-basket-api) 

#### What are the relevant pivotal tracker stories?
- [#150399214](https://www.pivotaltracker.com/story/show/150399214)
